src/plugins/regexdispatcher/README.md

```diff
@@ -0,0 +1,36 @@
+- infos = Information about the regexdispatcher plugin is in keys below
+- infos/author = e1528532 <e1528532@libelektra.org>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = 
+- infos/recommends =
+- infos/placements = presetstorage
+- infos/status = maintained experimental
+- infos/metadata =
+- infos/description = a plugin which generates regex-representations of specification keywords
+
+## Introduction
+
+A plugin which generates regex-representations of specification keywords for the 
+typechecker plugin before saving a configuration specification. Intended to be 
+mounted before the typechecker plugin.
+
+## Usage
+
+In order to generate regex representations for different specification keywords, mount a configuration specification along with this plugin. 
+
+`kdb mount <specification> spec/<path> <storage plugin to read the specification> regexdispatcher typechecker`
```

Comment:
Why not let the typechecker depend on the regexdispatcher? Or is it usful to use the typechecker without regexdispatcher!

Please describe more clearly why the regexdispatcher is needed.

Reply:
i will. basically the typechecker can also work without the regexdispatcher for certain metadata like check/long or check/validation or similar which do not need an intermediate translation process.
However something like check/enum/#1=A, check/enum/#2 = B needs to be translated to the regex A|B, which the regexdispatcher handles to keep concerns separated.
Furthermore other people may implement similar plugins that translate their own metakeys to regexes (that don't necessarily have to be implemented in haskell) and so i concluded to keep those things separated from the typechecker plugin.

Reply:
Yes, I assumed something like that. But the docu does not say it.